charts/generic-service/values.schema.json

```diff
@@ -83,7 +83,6 @@
     },
     "env": {
       "type": "object",
-      "additionalProperties": {"type": "string"},
```

Comment:
We probably still want to prevent the usage of non-primitive types.

How about:

```json
      "additionalProperties": {"type": ["string", "number", "boolean"]}
```

Reply:
Good point! Although `quote` should also deal with objects and lists, it wouldn't really make sense. I've added this change 👍